apps/src/lib/client/rpc.rs

```diff
@@ -975,6 +976,63 @@ pub async fn query_voting_power(ctx: Context, args: args::QueryVotingPower) {
     println!("Total voting power: {}", total_voting_power);
 }
 
+/// Query PoS commssion rate
+pub async fn query_commission_rate(
+    ctx: Context,
+    args: args::QueryCommissionRate,
+) {
+    let epoch = match args.epoch {
+        Some(epoch) => epoch,
+        None => query_epoch(args.query.clone()).await,
+    };
+    let client = HttpClient::new(args.query.ledger_address).unwrap();
+
+    match args.validator {
+        Some(validator) => {
+            let validator = ctx.get(&validator);
+            let validator_commission_key =
```

Comment:
I think we could add a call to `is_validator` here to check first for a bit nicer error handling when the given address is not a validator

Reply:
Thx, sounds good.